packages/opentelemetry-cloud-trace-exporter/test/transform.test.ts

```diff
@@ -130,6 +130,7 @@ describe('transform', () => {
   });
 
   it('should drop unknown attribute types', () => {
+    // @ts-expect-error
```

Comment:
Why do we need this?

Reply:
The test is asserting that unsupported attribute types (like object here) are dropped, but it creates a TS error since it is not allowed. In this case, we expect the TS error but want to validate the behavior too